icepack.setup

```diff
@@ -395,16 +424,28 @@ echo "#hash = ${hash}" >> results.log
 echo "#hshs = ${shhash}" >> results.log
 echo "#hshu = ${hashuser}" >> results.log
 echo "#hshd = ${hashdate}" >> results.log
+echo "#suit = ${testsuite}" >> results.log
 echo "#date = ${cdate}" >> results.log
 echo "#time = ${ctime}" >> results.log
 echo "#mach = ${machine}" >> results.log
 echo "#user = ${user}" >> results.log
 echo "#vers = ${vers}" >> results.log
 echo "#------- " >> results.log
+EOF0
+
+cat >! ${tsdir}/report_codecov.csh << EOF0
```

Comment:
Maybe it would make sense to only generate ``report_codecov.csh`` if ``icepack.setup`` is invoked with `--codecov` (I.e. wrap this section in `if ($codecov == 1) then` ... `endif`).

The script ``report_codecov.csh`` is of no use if test reports are not generated.